### ISSUE TITLE
All headers do not point to document root

### ISSUE THREAD

**Issue by Orio-nn** [eIfo1/social-website#5]
All header functions do not point to the document root. Whenever passing a function like `RequireAuthentication()` to a file in a directory like `/messages/send/` it will only go back 1 directory and into the login page. In theory, this would work, but as I said it only goes back 1 directory, or however many `../`'s you put. So, if you were to do this in the message send file, you would get sent  to `/messages/login/` which is a directory that doesn't exist.

TLDR: Add document root to header functions.
